server/models/conversationSchema.js

```diff
@@ -16,7 +16,9 @@ const conversationSchema = new Schema(
         alumni: [{type: Schema.Types.ObjectId, ref: 'Alumni'}],
         messages: [messageSchema],
         seen: [{type: Boolean}],
-        dateCreated: {type: Date, default: moment()}, 
+        dateCreated: {type: Date, default: moment()},
+        //RETURN FIELD NOT KEPT IN DB
+        timeFromMessage: {type: String}
```

Comment:
comment is pretty self explanatory, but this field is a workaround for returning fields that don't need to be stored on the database

I looked into mongoose's virtuals, and while they have the right functionality they aren't passed in a response unless I go through and convert every document to JSON before sending the response